projects/or_scs/test/Mirza/SupplyChain/Tests/Service.hs

```diff
@@ -71,6 +71,17 @@ testServiceQueries = do
       res <- runAppM @_ @AppError scsContext $ insertGS1Event dummyTransactEvent
       res `shouldSatisfy` isLeft
 
+  describe "Association Event" $ do
+ 
+    it "Insert Association Event" $ \scsContext -> do
+      (evInfo, _) <- testAppM scsContext $ insertGS1Event dummyAssocEvent
+      (eventInfoEvent evInfo) `shouldBe` dummyAssocEvent
+
+--    it "Should not allow duplicate Transaction events" $ \scsContext -> do
```

Comment:
Did you mean to leave out this commented out code?

Reply:
Ta. fixed.